website/docs/d/projects.html.markdown

```diff
@@ -64,6 +64,11 @@ data "mongodbatlas_projects" "test" {
 * `teams.#.role_names` - Each string in the array represents a project role assigned to the team. Every user associated with the team inherits these roles. The [MongoDB Documentation](https://www.mongodb.com/docs/atlas/reference/user-roles/#organization-roles) describes the roles a user can have.
```

Comment:
in Example Usage section, suggest we add an example in above showing users how to use new limits parameter with data source project**s**

Reply:
Makes sense, added a limit definition in the example usage section. Just to keep in mind, for both project and projects data source all limits will be fetched (even if there are not user defined ones), not sure if this is worth mentioning explicitly in the documentation

Reply:
I think it's OK to leave out for now, when in doubt would follow Atlas Admin API docs and they don't mention either. https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Projects/operation/getProjectLimit